### DISCUSSION TITLE
PhotoPrism in docker geolocation maps&places timeout

### DISCUSSION THREAD

**Discussion by rolandvonwedel** [photoprism/photoprism#4068] [Q&A]
I installed Photoprism in docker running on debian and its working fine except that I get the following error when indexing the photos:

- 2024-02-12 07:41:38 ERRO photo: Get "https://places.photoprism.app/v1/location/80c2bcc5f1bc": dial tcp 5.9.32.79:443: i/o timeout (find location)
- 2024-02-12 07:41:38 ERRO places: Get "https://places.photoprism.app/v1/location/80c2bcc5f1bc": dial tcp 5.9.32.79:443: i/o timeout (http request failed)

DNS seems to be working fine as the correct IP address for https://places.photoprism.app is deployed. Also when I ssh as root into the NAS and perform a "wget https://places.photoprism.app/v1/location/80c2bcc5f1bc" the data is received.

Any idea how this can be fixed?

**Comment by lastzero** [comment]
Is this still happening? Could have been a temporary connection problem.

**Reply by rolandvonwedel** [reply]
Hi, yes happening constantly since I installed 4 days ago during indexing.

**Reply by lastzero** [reply]
Since you only tested the connection from your NAS and not from the application container, I recommend opening a terminal and trying again from there:
- https://docs.photoprism.app/getting-started/docker-compose/#opening-a-terminal

Also see:
- https://docs.photoprism.app/getting-started/troubleshooting/firewall/

**Reply by rolandvonwedel** [reply]
Running the command:  "docker compose exec -u 1000 photoprism photoprism places update"
I get the following output:

INFO[2024-02-13T20:13:23Z] config: make sure your server has enough swap configured to prevent restarts when there are memory usage spikes 
INFO[2024-02-13T20:13:23Z] Become a member today, support our mission and enjoy our member benefits! 💎 
INFO[2024-02-13T20:13:23Z] Visit https://www.photoprism.app/membership to learn more. 
ERRO[2024-02-13T20:13:23Z] Since running this command puts additional load on our infrastructure, we unfortunately can only offer it to members. 

Does this mean reverse geolocation is only available to members?

**Reply by lastzero** [reply]
I was assuming that you were going to test `wget https://places.photoprism.app/v1/location/80c2bcc5f1bc` and not the `photoprism places update` command?

As detailed in [our documentation](https://docs.photoprism.app/getting-started/#maps-places) and [the feature overview](https://www.photoprism.app/editions#compare) on our website, reverse geocoding is free for everyone. However, the batch update command that updates all existing data at once is for members only due to the load it creates and also since you won't need it to get started.

**Reply by rolandvonwedel** [reply]
Thanks, well understood! I registered as Member now, still having some problems to activate but I will address this via the direct support channel. Congrats ayway for the great product!!!!

**Reply by lastzero** [reply]
If you need help with the activation, you can email use at membership@photoprism.app :mailbox_with_mail:
